SonarQube.Bootstrapper/Program.cs

```diff
@@ -47,8 +48,14 @@ private static int preprocess(ILogger logger, IBootstrapperSettings settings, st
             Debug.Assert(!string.IsNullOrWhiteSpace(server), "Not expecting the server url to be null/empty");
             logger.LogMessage(Resources.INFO_ServerUrl, server);
 
-            IBuildAgentUpdater updater = new BuildAgentUpdater();
-            if (!updater.TryUpdate(server, downloadBinPath, logger))
+            BuildAgentUpdater updater = new BuildAgentUpdater(logger);
+            if (!updater.TryUpdate(server, downloadBinPath))
+            {
+                logger.LogError(Resources.ERROR_CouldNotFindIntegrationZip);
+                return 1;
+            }
+
+            if (!BuildAgentUpdater.CheckBootstrapperVersion(settings.SupportedBootstrapperVersionsFilePath, settings.BootstrapperVersion))
```

Comment:
The versions file should in a known location relative to where the downloaded files were unzipped. This doesn't appear to be what is being used.